climakitae/core/data_interface.py

```diff
@@ -1802,6 +1804,12 @@ def get_data(
     area_average: one of ["Yes","No"], optional
         Take an average over spatial domain?
         Default to "No".
+    latitude: None or tuple of float, optional
+        Tuple of valid latitude bounds
+        Default to None: spatial subset by shapefile
+    longitude: None or tuple of float, optional
+        Tuple of valid longitude bounds
+        Default to None: spatial subset by shapefile
```

Comment:
Is this `shapefile` from a potentially selected `cached_area`?

Reply:
I think the colon is a bit confusing here, could this be re-worded to be something along the lines of "Default to None, otherwise spatial subset is done through..."?